editor/components/post-title/index.js

```diff
@@ -99,12 +99,15 @@ class PostTitle extends Component {
 						'mod+shift+z': this.redirectHistory,
 					} }
 				>
+					<label htmlFor="post-title" className="screen-reader-text">
+						{ placeholder || __( 'Add title' ) }
```

Comment:
We should still want `decodeEntities` here.

Reply:
Ah ok, thought it was meant just for HTML attributes.